src/app/js/formats/streamgraph/StreamgraphIcon.svg

```diff
@@ -0,0 +1,2 @@
2l0,0l0,0l0.5-0.1  c0.6-0.2,1.3-0.4,1.9-0.4c0.3,0,0.6,0,0.9,0l0.4-0.1l0.1,0l0.1,0l1.2-0.1l1.2,0l0,0h1.3l1.2,0l1.2,0h0.1l0.1,0l1.2,0l1.3,0.1  l1.3,0.1l0,0l0,0l1.2,0.1l1.2,0.1l0.1,0l0.1,0l1.3,0.1l1.3,0.1l1.3,0.1l1.3,0.1l0,0l0,0l1.3,0.1l0,0l0,0l1.3,0.2l1.3,0.2l0.6,0.1  l2.2,0.3V40.1l-2.1-0.7c0,0-0.3-0.1-0.3-0.1l-1.3-0.7l0,0l0,0l-1.3-0.8L88.6,37l-1.3-0.7l-1.3-0.8l-1.3-0.7l-1.3-0.7l-1.2-0.7  l-1.2-0.6l0,0l0,0l-1.2-0.7l-1.2-0.6l-1.2-0.6L76,30.3l-1.2-0.5l-1.2-0.5L72.5,29l-1.2-0.4l-1.1-0.3l-1.1-0.2L68,27.9L67,27.8l-1,0  l-1.1,0l-1,0.1l-1,0.2l-1,0.3l-1,0.4l-1,0.5l-1,0.6l-1,0.6l-1,0.8l-1,0.8l-1,0.9L53.6,34l-1,1.1l-1,1.2l-1.1,1.3L49.4,39l-1.1,1.4  l-1.1,1.4l-1,1.5l-1.1,1.5l-1,1.5L43,47.7l0,0l0,0l-1,1.3l-1,1.3l0,0.1l0,0.1l-1,1.1l0,0l0,0l-1,1l0,0l0,0l-1,0.9l-0.1,0.1l-0.1,0.1  l-1,0.7l-0.1,0.1l-0.2,0.1l-1,0.4l-0.2,0.1l-0.2,0l-1,0.2l-0.2,0l-0.2,0l-1,0l-0.3,0l-0.3-0.1l-1-0.3L30.6,55l-0.2-0.1l-0.6-0.4  l0.4,0.8l0.9,1.9L32,59l0.9,1.6l0.9,1.5l0.9,1.2l0.9,1l0.8,0.8l0.8,0.6l0.8,0.4l0.8,0.3l0.7,0.2l0.8,0l0.8,0l0.8-0.1l0.9-0.2  l0.9-0.3l0.9-0.4l1-0.5l1-0.5l1.1-0.6l1.1-0.6l1.1-0.6l1.1-0.6l1.1-0.6l1.1-0.6l1.1-0.6l0,0l0,0l1.1-0.5l0,0l0,0l1.1-0.5l1.1-0.5  l0.1,0l0.1,0l1.1-0.4L60.4,58L60.4,58z"/><path d="M44.6,92.8l1.1-0.6l1-0.6l1.1-0.6l1.1-0.6l0,0l0,0L50,90l1.1-0.5l0,0l0,0l1.1-0.5l0,0l0,0l1.1-0.5l0,0l0,0  l1.1-0.5l0,0l0,0l1.1-0.4l0,0l0,0l1.1-0.4l0,0l0,0l1.1-0.3l0.1,0l0.1,0l1.1-0.3l0.1,0l0.1,0l1.1-0.2l1.1-0.2l0.1,0l0.1,0l1.2,0  l1.3,0l1.2-0.2l0,0l0,0l1.2,0.1l0.2,0c0.1,0,0.1,0,0.2,0l0.9,0l1.2,0l0,0l0,0l1.2,0.2l0,0l0,0l1.2,0.2l1.2,0.2l0,0l0,0l1.2,0.2  l0.1,0l0.1,0l1.2,0.3l1.2,0.3l0,0l0,0l1.2,0.3l1.2,0.3l0,0l0,0l1.3,0.4l1.3,0.4l0,0l0,0l1.3,0.4l1.3,0.4l1.3,0.4l0,0l0,0l1.2,0.5  l1.3,0.4l0,0l0,0l1.3,0.5l1.3,0.4l1.3,0.5l0,0l0,0l0.2,0.1c0,0,1.7,0.5,2.4,0.8v-6.5c-0.7-0.2-1.7-0.3-2.3-0.7l-0.1-0.1l-1.3-0.5  l-1.3-0.5l0,0l0,0l-1.3-0.5l-1.3-0.5L86,83.6l0,0l0,0l-1.3-0.5l-1.3-0.5l-1.3-0.5l-1.3-0.4l-1.2-0.4l-1.2-0.4l-1.2-0.4l-1.2-0.4  l-1.2-0.4l-1.2-0.3l-1.2-0.3l-1.2-0.3L70,78.6l-1.1-0.2l-1.1-0.2l-1.1-0.1l0,0l0,0l-1.1-0.1l-1.1,0l-1.1,0l-1,0l-1,0.1l-1.1,0.2  l-1.1,0.2l-1,0.2L57,79.2l-1.1,0.3l-1,0.4l-1.1,0.4l-1.1,0.4l-1.1,0.5l-1.1,0.5l-1,0.5l-1.1,0.5l0,0l0,0l-1.1,0.5l0,0l0,0l-1.1,0.5  l-1.1,0.5l0,0l0,0l-1.1,0.4l0,0l0,0l-1,0.4l-0.1,0l-0.1,0l-1,0.3l0,0l0,0l-1,0.2l-0.1,0l-0.1,0l-1,0.2l-0.1,0l-0.1,0l-1,0l-0.1,0  l-0.1,0l-1-0.1l-0.1,0l-0.1,0l-1-0.2l-0.1,0l-0.1,0l-1-0.4l-0.1,0l-0.1,0l-1-0.5l-0.1-0.1l-0.1-0.1l-1-0.7l-0.1-0.1L32,83.7l-1-0.8  l-0.1,0l-0.1,0l-1-1l0,0l0,0l-1-1.1l0,0l0,0l-1-1.2l0,0l0,0l-1-1.3l-1-1.3L25,75.3l-0.9-1.2L23.2,73l-0.8-0.9l-0.7-0.6l-0.4-0.2  l-0.1,0L21,71.3l-0.5,0.4l-0.7,1.1L19,74.5l-0.9,2.3l-0.9,2.8l-0.9,3.2l-0.4,1.6l0.5-1l0.1-0.1l0.1-0.1l0.9-1.2l0.2-0.2l0.2-0.2  l0.9-0.7l0.3-0.2l0.3-0.1l0.9-0.3l0.5-0.1l0.5,0.1l0.9,0.2l0.3,0.1l0.2,0.1l0.9,0.5l0.2,0.1l0.2,0.1l0.9,0.9l0.1,0.1l0.1,0.1  l0.9,1.1l0,0.1l0,0.1l0.9,1.2l1,1.3l0,0l0,0l0.9,1.4l0.9,1.3l0.9,1.2l0.9,1.1l0.9,1l0.9,0.8l0.8,0.7l0.8,0.5l0.8,0.4l0.8,0.3  l0.8,0.2l0.7,0l0.8-0.1l0.9-0.1l0.9-0.3l0.9-0.3l0.9-0.4l1-0.5L44.6,92.8z"/>
```

Comment:
Are you importing the `StreamgraphIcon.svg` somewhere ?
If no, you can delete it.

Reply:
it's not, I took the bubble chart as example which do the same. I though you was putting the svg to keep a trace of the origin file but no problems, will delete it.